### PR TITLE
Update obsolete links in README and PR template

### PR DESCRIPTION
**Pull request by VojtechVitek**
**What this PR does / why we need it**:
Updates obsolete links in README and Pull Request template. All the links now lead directly to kubernetes/community repository, without the intermediate "moved to" pages.

### CODE REVIEW

File: docs/getting-started-guides/docker.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-This file has moved to: https://github.com/kubernetes/kubernetes/blob/master/docs/devel/local-cluster/docker.md
+This file has moved to [https://github.com/kubernetes/community/blob/master/contributors/devel/local-cluster/docker.md](https://github.com/kubernetes/community/blob/master/contributors/devel/local-cluster/docker.md)
```
Comment (cblecker) [inline docs/getting-started-guides/docker.md]: This doesn't need the `[]()` format as it's a link.

**Review by cblecker — COMMENTED** [review]
one nit, otherwise looks good.

**Comment by thockin** [comment]
@k8s-bot ok to test

**Comment by thockin** [comment]
manual

**Pull request merged**